ios/REAModule.mm

```diff
@@ -51,6 +51,29 @@ @implementation REAModule {
 
 RCT_EXPORT_MODULE(ReanimatedModule);
 
+// This counts how many instances of REAModule are present. Thanks to this we
+// can easily detect if there is a memory leak (ex. due to a retain cycle).
+static int instanceCounter = 0;
```

Comment:
Can we rename it to `instanceCounterREAModule` and also implement an analogous mechanism for `NativeReanimatedModule`?

Reply:
Sure, I'll also implement something similar for Android